app/src/main/java/org/dhis2/usescases/login/LoginActivity.kt

```diff
@@ -154,22 +161,13 @@ class LoginActivity : ActivityGlobalAbstract(), LoginContracts.View {
                 }
             }
         }
-
-        fun accountIntentResult(
-            serverUrl: String?,
-            userName: String?,
-            wasAccountClicked: Boolean,
-        ): Intent = Intent().apply {
-            serverUrl?.let { putExtra(RESULT_ACCOUNT_SERVER, serverUrl) }
-            userName?.let { putExtra(RESULT_ACCOUNT_USERNAME, userName) }
-            putExtra(FROM_SPLASH, false)
-            putExtra(RESULT_ACCOUNT_CLICKED, wasAccountClicked)
-        }
     }
 
     @SuppressLint("ClickableViewAccessibility")
     override fun onCreate(savedInstanceState: Bundle?) {
-        setTheme(R.style.LoginTheme)
+        enableEdgeToEdge(
+            statusBarStyle = SystemBarStyle.dark(Color.WHITE),
```

Comment:
why don't you use surface bright instead of direct White? to use our design system color palette

Reply:
This is not compose. So I can only access android graphic colors